activesupport/CHANGELOG.md

```diff
@@ -1,2 +1,7 @@
+*   Add instrumentation for ActionController::Live#send_stream
+
+    Allows subscribing to `send_stream` events. The event payload contains the filename, disposition, and type.
+
+    *Hannah Ramadan*
```

Comment:
One small nitpick. As this is a change in actionpack, I think it should be added to the actionpack changelog instead.

Reply:
@p8  Made this change ◡̈